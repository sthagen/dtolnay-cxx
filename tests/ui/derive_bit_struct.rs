#[cxx::bridge]
mod ffi {
    #[derive(BitAnd, BitOr, BitXor)]
    struct Struct {
        x: i32,
    }
}

fn main() {}
