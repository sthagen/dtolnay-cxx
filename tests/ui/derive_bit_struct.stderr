error: derive(BitAnd) is currently only supported on enums, not structs
 --> tests/ui/derive_bit_struct.rs:3:14
  |
3 |     #[derive(BitAnd, BitOr, BitXor)]
  |              ^^^^^^

error: derive(BitOr) is currently only supported on enums, not structs
 --> tests/ui/derive_bit_struct.rs:3:22
  |
3 |     #[derive(BitAnd, BitOr, BitXor)]
  |                      ^^^^^

error: derive(BitXor) is currently only supported on enums, not structs
 --> tests/ui/derive_bit_struct.rs:3:29
  |
3 |     #[derive(BitAnd, BitOr, BitXor)]
  |                             ^^^^^^
